src/shared.ts

```diff
@@ -59,6 +59,12 @@ export const loadScript = (): Promise<StripeConnectWrapper | null> => {
   }
 
   stripePromise = new Promise((resolve, reject) => {
+    if (typeof window === "undefined") {
+      reject(
+        "ConnectJs won't load when rendering code in the server - it can only be loaded on a browser. This error is expected when loading ConnectJS in SSR environments, like NextJS. It will have no impact in the UI, however if you wish to avoid it, you can switch to the `pure` version of the connect.js loader: https://github.com/stripe/connect-js#importing-loadconnect-without-side-effects."
```

Comment:
Add capitalization for ConnectJs

```suggestion
        "ConnectJS won't load when rendering code in the server - it can only be loaded on a browser. This error is expected when loading ConnectJS in SSR environments, like NextJS. It will have no impact in the UI, however if you wish to avoid it, you can switch to the `pure` version of the connect.js loader: https://github.com/stripe/connect-js#importing-loadconnect-without-side-effects."
```